lib/genome/groupers/drug_grouper.rb

```diff
@@ -178,10 +183,7 @@ def normalizer_matches_for_term(term)
         if term_to_matches_dict.has_key? term
           return term_to_matches_dict[term]
         end
-        uri = URI.parse(normalizer_url).tap do |u|
-          u.query = URI.encode_www_form( { q: ERB::Util.url_encode(term) } )
-          u.port = 8000
-        end
+        uri.query = URI.encode_www_form( { q: ERB::Util.url_encode(term) } )
         res = Net::HTTP.get_response(uri)
```

Comment:
This might work but I'm not actually 100% sure to be honest. I see that you're setting a persistent connection in the `URI` construction at the beginning but I'm unsure if this way of invoking it will actually use that or not. Have you tested it out? If not, I can give it a go.